viewing_party/party.py

```diff
@@ -1,23 +1,125 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    if not title or not genre or not rating:
+        return None
+    else:
+        movie = {}
+        movie["title"] = title
+        movie["genre"] = genre
+        movie["rating"] = rating
+    return movie
+
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+    return user_data
+
+def watch_movie(user_data, movie_title):
+    movie = next((item for item in user_data["watchlist"] if item["title"] == movie_title), None)

```

Comment:
Solid use of list comprehension when we haven't covered it yet! Returning a list here and calling `next` on it certainly works, but you could also do it with a for loop like
```python
    for movie in user_data["watchlist"]:
        if title == movie["title"]:
            # append and remove from watched and watchlist accordingly
```
Also, while it's a good thought to use `add_to_watched` and it absolutely works, it's not ideal to call a function which has a return value that you're not going to use. `add_to_watched` correctly mutates the `user_data` but I think in this case it's clearer to simply append it to the user data watchlist inline:
```python
user_data["watched"].append(movie)
```